src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process/WaitingProcessOutputLineListener.java

```diff
@@ -18,38 +18,36 @@
 
 import com.google.cloud.tools.appengine.cloudsdk.process.ProcessExitListener;
 import com.google.cloud.tools.appengine.cloudsdk.process.ProcessOutputLineListener;
-
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
 /**
  * Provides a mechanism to wait for a successful start of a process by monitoring the process output
  * and checking for a specific message in it.
  */
-public class WaitingProcessOutputLineListener implements ProcessOutputLineListener,
-    ProcessExitListener {
+public class WaitingProcessOutputLineListener
+    implements ProcessOutputLineListener, ProcessExitListener {
   private final String message;
   private final int timeoutSeconds;
   private CountDownLatch waitLatch;
   private volatile boolean exited;
 
   /**
-   * @param message        The message to look for in the output of the process to consider it to be
-   *                       successfully started. If the message is not seen within the specified
-   *                       timeout, a {@link ProcessRunnerException} will be thrown. The message is
-   *                       assumed to be a regular expression.
+   * Constructor.
```

Comment:
Can't say I'm thrilled with some of these autofixes.

Reply:
Hah, that one's one me, since there was no summary, I threw one in, I can make it better.